pkgs/servers/default.nix

```diff
@@ -1,3 +1,5 @@
 { callPackage, ... }:
 
-{ } // (callPackage ./shortbin.nix { })
+{
+  waterfall = callPackage ./waterfall { };
```

Comment:
Can you add version suffix?

Reply:
Waterfall is minecraft version independent, so I think it is not required.

Reply:
Fixed